### PR TITLE
fixed sonar issues on master branch related to 2249

### PR DESCRIPTION
**Pull request by JamesCutts00**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRL-2449

### Change description ###
Fixed all issues on the master branch relating to ticket 2449.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/main/steps/common/keep-details-private/KeepDetailsPrivateGetController.ts [hunk 1]
```diff
@@ -9,6 +9,33 @@ import { RESPONDENT_DETAILS_KNOWN } from '../../urls';
...
+          applicant?.value?.user?.idamId === req.session?.user.id &&
+          applicant?.value?.response &&
+          applicant?.value?.response?.keepDetailsPrivate &&
+          applicant?.value?.response?.keepDetailsPrivate?.confidentiality
+        ) {
+          Object.assign(req.session.userCase, getKeepYourDetailsPrivate(applicant.value, req));
+        }
+      });
+    }
+  }
```
Comment (AlokDatta) [inline src/main/steps/common/keep-details-private/KeepDetailsPrivateGetController.ts]: Why added this peice of code?

File: src/main/steps/common/keep-details-private/KeepDetailsPrivateGetController.ts [hunk 2]
```diff
@@ -9,6 +9,32 @@ import { RESPONDENT_DETAILS_KNOWN } from '../../urls';
...
```
Comment (AlokDatta) [inline src/main/steps/common/keep-details-private/KeepDetailsPrivateGetController.ts:36]: why added this code?

File: src/main/steps/common/keep-details-private/KeepDetailsPrivatePostController.ts [hunk 3]
```diff
@@ -22,6 +21,41 @@ export class KeepDetailsPrivatePostController extends PostController<AnyObject>
...
+
+  public async postFL401(req: AppRequest<AnyObject>) {
+    if (req.url.includes('respondent')) {
+      if (req.session.userCase?.respondentsFL401?.user?.idamId === req.session?.user.id) {
+        Object.assign(
+          req.session.userCase.respondentsFL401,
+          setKeepYourDetailsPrivate(req.session.userCase.respondentsFL401, req)
+        );
+      }
+    } else {
```
Comment (AlokDatta) [inline src/main/steps/common/keep-details-private/KeepDetailsPrivatePostController.ts:49]: throwing lint error

Comment (JamesCutts00) [reply src/main/steps/common/keep-details-private/KeepDetailsPrivatePostController.ts]: Fixed the Lint errors.

**Review by AlokDatta — APPROVED** [review]

**Pull request merged**